tests/integration/ovs_test.py

```diff
@@ -214,7 +214,7 @@ def test_nm_ovs_plugin_missing():
 
 
 def test_ovs_service_missing():
```

Comment:
Let's xfail for k8s this test and fix it later on at different PR stopping ovs servie at nodes and starting it up at teardown.

Reply:
No need. New patch is using `mount -o bind /dev/null /var/run/openvswitch/db.sock` to simulate the OVSDB daemon stopped.